streams/src/test/java/org/apache/kafka/streams/integration/StreamsUncaughtExceptionHandlerIntegrationTest.java

```diff
@@ -157,7 +158,8 @@ public void shouldShutdownThreadUsingOldHandler() throws Exception {
 
     @Test
     public void shouldShutdownClient() throws Exception {
-        try (final KafkaStreams kafkaStreams = new KafkaStreams(builder.build(), properties)) {
+        properties = basicProps();
+        try (final KafkaStreams kafkaStreams = new KafkaStreams(builder.build(), basicProps())) {
```

Comment:
why creating `basicProps()` again?

Reply:
My mistake, thanks for pointing that out.